todo.sh

```diff
@@ -101,7 +113,7 @@ while test $# -gt 0; do
       exit 0
       ;;
     --this-dir*)
```

Comment:
This is actually a boolean flag so this could be `--this-dir)`, but that's my mistake :)